amida-tech/amida-auth-microservice
# Hotfix ser 274 ser 251

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
@B3rry found a bug
```
> docker run amidatech/auth-service
...
Error: Cannot find module 'source-map-support/register'`
...
```

#### Related JIRA tickets:
#### How should this be manually tested?
```
> docker run amidatech/auth-service:2.7.1-rc.0
...
/app/src/config/config.js:129
    throw new Error(`Config validation error: ${error.message}`);
          ^
Error: Config validation error: child "AUTH_SERVICE_REGISTRAR_SCOPES" fails because ["AUTH_SERVICE_REGISTRAR_SCOPES" is required]
```
...cause no config (this is expected)

Then with the config...
```
> docker run -p 4000:4000 -v <ABS PATH>.env:/app/.env:ro amidatech/auth-service:2.7.1-rc.0
yarn run v1.12.3
$ node dist/index.js
```

Can also use `amidatech/auth-service:dev` since hotfix are also applied to dev. _Just remember to delete the stale image_

#### Any background context you want to provide?
#### Screenshots (if appropriate):
